src/com_tjvendors/admin/models/vendors.php

```diff
@@ -319,9 +319,8 @@ public function setItemState($items, $state, $client)
 			}
 		}
 
-		$dispatcher = JDispatcher::getInstance();
 		PluginHelper::importPlugin('tjvendors');
-		$dispatcher->trigger('tjVendorsOnAfterVendorStateChange', array($items, $state, $client));
+		Factory::getApplication()->triggerEvent('tjVendorsOnAfterVendorStateChange', array($items, $state, $client));
```

Comment:
dont triggers needs to be starting as `on`

Reply:
done